lib/documents.js

```diff
@@ -547,6 +547,14 @@ function Document(id, uid, type, href, tags, slugs, data) {
    */
   this.data = data;
   /**
+   * The first publication date of the document
+   */
+  this.firstPublicationDate = new Date(firstPublicationDate);
```

Comment:
Can these dates be null? I think so, right?

If so, passing null to `new Date(..)` will return a date with year 1970. Not too sure if we want to add some extra check?